external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaContinuousSourceSuite.scala

```diff
@@ -169,6 +169,60 @@ class KafkaContinuousSourceSuite extends KafkaSourceSuiteBase with KafkaContinuo
       }
     }
   }
+
+  Seq("continuous", "micro-batch").foreach { mode => {
+    val trigger = mode match {
+      case "continuous" => ContinuousTrigger(100)
+      case "micro-batch" => ProcessingTime(1000)
+    }
+    test(s"SPARK-27494: read kafka record containing null key/values in $mode mode") {
```

Comment:
shall we move it to `KafkaSourceSuiteBase`?

Reply:
+1 updated